app/src/main/res/layout/hints_and_solution_fragment.xml

```diff
@@ -27,7 +27,7 @@
       <androidx.appcompat.widget.Toolbar
         android:id="@+id/hints_and_solution_toolbar"
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
+        android:layout_height="wrap_content"
```

Comment:
@adizcode Instead of `wrap_content` you need to declare the height in dimens.xml.